packages/react-menu/e2e/Menu.e2e.ts

```diff
@@ -39,11 +40,7 @@ describe('MenuTrigger', () => {
 
 describe('Custom Trigger', () => {
   it('should open menu when clicked', () => {
-    cy.visitStory('Menu', customTriggerStory)
-      .contains('Custom Trigger')
-      .click()
-      .get(menuSelector)
-      .should('be.visible');
+    cy.visitStory('Menu', customTriggerStory).contains('Custom Trigger').click().get(menuSelector).should('be.visible');
```

Comment:
unfortunately this still fits in the line limit so prettier wants to 'fix' it 🤦‍♀️